Metadata.yaml

```diff
@@ -0,0 +1,246 @@
+        to Developers. These are guidelines and being above or below these
+        arbitrary cutoffs does not automatically include or exclude you. E.g.
+        someone doing complicated low-level code will likely become a Developer
+        with far fewer lines of code contributed.
+
+      This author list is ordered by last name.
+    List:
+      - Name: Armaza, Cristóbal
+        Orcid: 0000-0002-1791-0743
+        Affiliations: [*cornell]
+        GitHub: carmaza
+      - Name: Bonilla, Gabriel S.
+        Orcid: 0000-0003-4502-528X
+        Affiliations: [*cornell]
+        GitHub: gsb76
+      - Name: Hébert, François
+        Orcid: 0000-0001-9009-6955
+        Affiliations: [*caltech]
+        GitHub: fmahebert
+      - Name: Kumar, Prayush
+        Orcid: 0000-0001-5523-4603
+        Affiliations: [*cornell]
+        GitHub: prayush
+      - Name: Lovelace, Geoffrey
+        Orcid: 0000-0002-7084-1070
+        Affiliations: [*fullerton]
+        GitHub: geoffrey4444
+      - Name: Moxon, Jordan
+        Orcid: 0000-0001-9891-8677
+        Affiliations: [*caltech]
+        GitHub: moxcodes
+      - Name: O'Shea, Eamonn
+        Affiliations: [*cornell]
+        GitHub: osheamonn
+      - Name: Scheel, Mark A.
+        Orcid: 0000-0001-6656-9134
+        Affiliations: [*caltech]
+        GitHub: markscheel
+      - Name: Teukolsky, Saul A.
+        Orcid: 0000-0001-9765-4526
+        Affiliations: [*caltech, *cornell]
+        GitHub: teukolsky
+
+  Contributors:
+    Description: |
+      Anyone who has contributed to SpECTRE will be added to this list and
+      remain a Contributor indefinitely (unless they wish to be removed).
+
+      Guidelines on becoming a Contributor:
+
+      - See our guidelines in `docs/CONTRIBUTING.md` for information on how to
+        get involved with SpECTRE development.
+      - Having any PR merged qualifies for this list. If multiple people worked
+        on the PR together, e.g. several undergraduate students contributing to
+        a single PR, all persons involved will become a Contributor.
+
+      This author list is ordered by last name.
+
+    List:
+      - Name: Anantpurkar, Isha
+        Orcid: 0000-0002-5814-4109
+        Affiliations: [*cornell]
+        GitHub: isha1810
+      - Name: Foucart, Francois
+        Orcid: 0000-0003-4617-4738
+        Affiliations: [*new_hampshire]
+        GitHub: ffoucart
+      - Name: Giesler, Matthew
+        Orcid: 0000-0003-2300-893X
+        Affiliations: [*cornell]
+        GitHub: mgiesler
+      - Name: Iozzo, Dante A. B.
+        Orcid: 0000-0002-7244-1900
+        Affiliations: [*cornell]
+        GitHub: "10220"
+      - Name: Legred, Isaac
+        Orcid: 0000-0002-9523-9617
+        Affiliations: [*caltech]
+        GitHub: isaaclegred
+      - Name: Li, Dongjun
+        Orcid: 0000-0002-1962-680X
+        Affiliations: [*caltech]
+        GitHub: dongjun826
+      - Name: Macedo, Alex
```

Comment:
This is quite a pedantic comment, but if possible, could "Alex" be changed to "Alexandra" to reflect my full first name? Wasn't sure if an edit to the contributors' information belonged in this PR or in our own PR after this release.

Reply:
That's not pedantic at all, of course I'll change that!

Reply:
Piggy backing on what Alex said, would it be okay if we change "Teresa" to my full name "Teresita" as well? :)

Reply:
Of course, should be fixed now!

Reply:
Thank you Nils! :)

Reply:
I'll echo what @nilsleiffischer: asking us to spell your name the way you want it spelled is not at all pedantic :)

Reply:
Thank you!